src/utils/appLinking.js

```diff
@@ -0,0 +1,147 @@
+import { Linking } from "react-native";
+import { Alert } from "../components";
+import {
+	team,
+	general,
+	announcements,
+	fundraisingCommittee,
+	motorshpe,
+	projects20_21,
+	shpejr,
+	mentorshpe20_21,
+	computereng,
+	projectleads,
+	internationalStudents,
+	linkedinDrop
+} from "react-native-dotenv";
```

Comment:
Are these not being imported from the YAML file now?

Reply:
no. I could figure out yaml. There are been imported from dotenv file.